src/System.Security.AccessControl/tests/System.Security.AccessControl.Tests.csproj

```diff
@@ -1,8 +1,9 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <ProjectGuid>{70FAC855-CAC6-4523-8477-880548D58A1B}</ProjectGuid>
     <UnsupportedPlatforms>Linux;NetBSD;OSX</UnsupportedPlatforms>
     <Configurations>netstandard-Windows_NT-Debug;netstandard-Windows_NT-Release</Configurations>
+    <DefineConstants>$(DefineConstants);ACCESSCONTROL_ASSEMBLY</DefineConstants>
```

Comment:
Why would the tests declare this?  They don't seem to be including the shared source that uses this value.